### PR TITLE
test: DXJ-576 updating the deal by adding a module

### PR DESCRIPTION
**Pull request by vasd85**

### CODE REVIEW

File: test/schemasAndValidation.ts
Comment (akim-bow) [inline test/schemasAndValidation.ts:1]: I don't really like this filename.
I suggest to create a folder `validators` and add in there files according to each validator, e.g. worker validator, service validator.

File: test/sharedSteps.ts [hunk 1]
```diff
@@ -61,6 +102,50 @@ export async function getFluenceConfig(cwd: string) {
...
+  const pathToServiceDir = getServiceDirPath(cwd, serviceName);
+
+  const serviceConfig = await initServiceConfig(
+    relative(cwd, pathToServiceDir),
+    cwd,
+  );
+
+  assert(
+    serviceConfig !== null,
+    `we create a service at ${pathToServiceDir} above - so the config is expected to exist`,
```
Comment (akim-bow) [inline test/sharedSteps.ts:115]:
```suggestion
    `the service, owning this config, must be created at the path ${pathToServiceDir}`,
```

File: test/sharedSteps.ts [hunk 2]
```diff
@@ -255,65 +340,44 @@ export async function waitUntilShowSubnetReturnsExpected(
...
-        Object.prototype.hasOwnProperty.call(config.services, serviceName),
-      `${serviceName} is expected to be in services property of ${config.$getPath()} after ${serviceName} is added to it`,
-    );
+      const runDeployedServicesResult = JSON.parse(result);
 
-    return config;
-  };
+      if (!validateDeployedServicesAnswerSchema(runDeployedServicesResult)) {
+        throw new Error(
+          `result of running ${RUN_DEPLOYED_SERVICES_FUNCTION_CALL} aqua function is expected to be an array of Answer, but it is: ${result}`,
```
Comment (akim-bow) [inline test/sharedSteps.ts:362]:
```suggestion
          `result of running ${RUN_DEPLOYED_SERVICES_FUNCTION_CALL} aqua function is expected to be an array of Answer, but actual result is: ${result}`,
```

**Review by shamsartem — APPROVED** [review]
looks fine in general, I think

**Pull request merged**